package.json

```diff
@@ -8,5 +8,10 @@
     "start": "node server.js"
   },
   "author": "Aditya Bansal",
-  "license": "ISC"
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
I would place dependencies under the `dependencies` key as these dependencies are required for running it and not just developing.